tests/test_typescript_features.py

```
@@ -0,0 +1,524 @@
+        pytest.fail(f"tldr command failed: {result.stderr}")
+    try:
+        return json.loads(result.stdout)
+    except json.JSONDecodeError:
+        # Some commands return non-JSON output
+        return {"raw_output": result.stdout}
+
+
+@pytest.fixture
+def temp_ts_file():
+    """Create a temporary TypeScript file."""
+    def _create(content: str, filename: str = "test.ts"):
```

Ruff (0.14.11)

Missing return type annotation for private function `_create`

(ANN202)